src/lua/lua_tcp.c

```diff
@@ -92,6 +92,61 @@ rspamd_config:register_symbol({
 
 LUA_FUNCTION_DEF (tcp, request);
 
+/***
+ * @method connect_sync()
```

Comment:
`lua_tcp.connect_sync` - it is a function and it should reference the module as well.